run_tests.sh

```diff
@@ -3,8 +3,8 @@
 pip3 install .
 mv byudml tmp_byudml
 reset
-python3 primitive_jsons/generate_primitive_jsons.py
-python3 pipelines/generate_pipelines.py
+python3 submission/pipelines/generate_pipelines.py
+python3 submission/primitive_jsons/generate_primitive_jsons.py
```

Comment:
I thought I had made a comment about this already. These lines should be switched back. Generating the primitive json runs some validation code that should pass before using our primitives in a pipeline.

Reply:
You made this comment in the next PR, which is a branch off of this one.  I'll change the order back in this one